app/src/test/java/com/example/parkingsystem/ParkingSizeTest.java

```diff
@@ -0,0 +1,62 @@
+package com.example.parkingsystem;
+
+import android.util.Log;
+
+import com.example.parkingsystem.mvp.model.ParkingModel;
+import com.example.parkingsystem.mvp.presenter.ParkingPresenter;
+import com.example.parkingsystem.mvp.view.ParkingView;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.MockedStatic;
+
+import static org.mockito.Mockito.doThrow;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.mockStatic;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+public class ParkingSizeTest {
+
+    private ParkingPresenter presenter;
+    private ParkingModel model;
+    private ParkingView view;
+    private MockedStatic<Log> logMockedStatic;
+
+    @Before
+    public void setup() {
+        model = mock(ParkingModel.class);
+        view = mock(ParkingView.class);
+        presenter = new ParkingPresenter(model, view);
+        logMockedStatic = mockStatic(Log.class);
+    }
+
+    @Test
+    public void onParkingSizeCreationButtonPressed_validSize() {
+        //Given
+        when(view.getSizeSubmitted()).thenReturn("15");
+        when(model.getParkingSize()).thenReturn(15);
+        //When
+        presenter.onParkingSizeCreationButtonPressed();
+        //Then
+        verify(model).setParkingSize("15");
+        verify(view).showParkingSize(15);
+    }
+
+    @Test
+    public void onParkingSizeCreationButtonPressed_invalidSize() {
```

Comment:
_methodName_condition_result_

I think you are missing the result in the name

Reply:
names updated!